core/components/commerce_timeslots/src/Admin/Schedule/Grid.php

```diff
@@ -87,8 +88,50 @@ public function prepareItem(\ctsSchedule $schedule)
         $editLink = $this->adapter->makeAdminUrl('timeslots/schedule/edit', ['id' => $schedule->get('id')]);
         $item['name'] = '<a href="' . $editLink . '" class="commerce-ajax-modal">' . $this->encode($item['name']) . '</a>';
         $item['name'] .= ' <nobr style="color: #6a6a6a;">(#' . $item['id'] . ')</nobr>';
-        if ($schedule->get('default')) {
-            $item['name'] .= '<br><span style="color: #6a6a6a; font-size: 9px;">*' . $this->adapter->lexicon('commerce_timeslots.default') . '</span>';
+
+        // Display the days this schedule is assigned to automatically
+        if ($schedule->get('repeat')) {
+            $item['name'] .= '<div style="margin:16px 0 8px 0; font-size:11px; color:#6a6a6a;"><i class="icon icon-calendar"></i> Scheduled Days</div>';
+
+            $methods = $this->adapter->getCollection(\comShippingMethod::class, [
+                'class_key' => \TimeSlotsShippingMethod::class
```

Comment:
Same as above, soft-deleting shipping methods so this should have a `removed => false` check